README.md

```diff
@@ -68,6 +70,27 @@ yarn install # only needed once
 yarn watch
 ```
 
+### MacOS
+
+On Mac OS we were unable to build the extension using the default clang installation.
+As a workaround we installed llvm using brew and used it to build the web extension.
+
+Install llvm
+
+```bash
+brew install llvm
+```
+
+Run the extension
+
+```bash
+cd extension;
+CC=/usr/local/opt/llvm/bin/clang AR=/usr/local/opt/llvm/bin/llvm-ar yarn install
+CC=/usr/local/opt/llvm/bin/clang AR=/usr/local/opt/llvm/bin/llvm-ar yarn watch
+```
+
+## Running the Web App
```

Comment:
This chapter is confusing and should be removed.

Maybe we should generally not introduce these chapters. I propose to mention the MacOs setup inline or at the bottom. 

yarn start is part of `Run the Web Extension` - it is the command that actually starts a browser instance that has the web extension installed.

Reply:
Ill get rid of this chapter but I think we still need the other ones. It doesn't seem understandable otherwise

Reply:
I pushed a few fixes including one that fixes the overall chapter structure.